### ISSUE TITLE
Self-extracting archive: talk about deps, exec rights and locations

### ISSUE THREAD

**Issue by DjSlash** [kaa-ching/tbe#59]
The self-extracting archive needs some more work:

 * [ ] discuss dependencies
 * [ ] discuss executable rights
 * [ ] discuss where to install

**Comment by kaa-ching** [comment]
If I'm not mistaken, this issue was created because you took the above text from the webpage http://kaa-ching.github.io/tbe/install.html and put it into a ticket, right?

**Comment by DjSlash** [comment]
That's right. Maybe re-wording is in place. Feel free to edit my initial comment. ;-)
